models/surveys.go

```diff
@@ -418,6 +435,26 @@ func (api *API) getSurveyID(surveyID string) error {
 	return api.GetSurveyIDStmt.QueryRow(surveyID).Scan(&id)
 }
 
+func isUuid(id string, idName string, w http.ResponseWriter) bool {
```

Comment:
Not sure if `isUUID` is the most expressive name as the function does more than a boolean test as it implies.

Reply:
any suggestions? because it currently does return true or false depending on if it is a UUID or not. validateUuid? isValidUuid?

Reply:
codacy has raised an issue with this func name - it should be "isUUID", see this note on initialisms https://github.com/golang/go/wiki/CodeReviewComments#initialisms